src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/default-transcripts-list-item/DefaultTranscriptListItem.test.tsx

```diff
@@ -36,28 +39,40 @@ jest.mock(
   () => () => <div>UnsplicedTranscript</div>
 );
 
+const toggleTranscriptInfo = jest.fn();
+
 describe('<DefaultTranscriptListItem />', () => {
   let wrapper: any;
 
-  beforeEach(() => {
-    const props = {
-      gene: createGene(),
-      transcript: createTranscript(),
-      rulerTicks: createRulerTicks()
-    };
+  const defaultProps = {
+    gene: createGene(),
+    transcript: createTranscript(),
+    rulerTicks: createRulerTicks(),
+    expandTranscript: false,
+    expandDownload: false,
+    toggleTranscriptInfo: toggleTranscriptInfo
+  };
 
-    wrapper = mount(<DefaultTranscriptListItem {...props} />);
-  });
+  const renderComponent = (props?: Partial<DefaultTranscriptListItemProps>) =>
+    mount(<DefaultTranscriptListItem {...defaultProps} {...props} />);
 
   it('displays unspliced transcript', () => {
+    wrapper = renderComponent();
     expect(wrapper.exists(UnsplicedTranscript)).toBe(true);
   });
 
-  it('toggles transcript item info', () => {
+  it('toggles transcript item info onClick', () => {
+    wrapper = renderComponent();
     wrapper.find('.middle').simulate('click');
-    expect(wrapper.exists(TranscriptsListItemInfo)).toBe(true);
+    expect(toggleTranscriptInfo).toBeCalled();
 
     wrapper.find('.right').simulate('click');
-    expect(wrapper.exists(TranscriptsListItemInfo)).toBe(false);
+    expect(toggleTranscriptInfo).toBeCalled();
```

Comment:
this line doesn't really check anything. Even if the `click` on the previous line did not trigger `toggleTranscriptInfo`, it would still test positive with `toBeCalled`, because we have not reset the mock since line 67.

Reply:
Added the block to reset the mock